packages/api/tests/models/rules/test-rules-model.js

```diff
@@ -196,6 +196,19 @@ test.serial('Creating an invalid rule does not create workflow triggers', async
   }
 });
 
+test('Creating a rule with a createdAt and updatedAt date assigns the correct values to createdAt and provides a new updatedAt', async (t) => {
+  const newRule = fakeRuleFactoryV2();
+  const currentDate = new Date('1/2/2030').getTime();
+  newRule.workflow = workflow;
+
+  delete newRule.collection;
+  delete newRule.provider;
+
+  const rule = await rulesModel.create(newRule, currentDate, currentDate);
+  t.is(rule.createdAt, currentDate);
+  t.not(rule.updatedAt, currentDate);
```

Comment:
Shouldn't the `updatedAt` value also be set to the `currentDate`? Unless I'm misunderstanding

Reply:
No, it gets updated to a more recent date later down the code chain: https://github.com/nasa/cumulus/blob/feature-rds/packages/api/models/base.js#L298

Reply:
Ah right. Though I wonder in that case if there's a point in doing tihs?

https://github.com/nasa/cumulus/pull/2012/files#diff-2a37aa7b44507f6480bc0edc75d3aa7e70031468d98c108f8aef958a2098ef91R246

Reply:
Hm that is a good point. I suppose it doesn't appear to be a point then, because we can just pass `Date.now()` and then it will get updated later anyway.